### DISCUSSION TITLE
What’s the retention period for backups?

### DISCUSSION THREAD

**Discussion by emoido** [ChistaDATA/Cloud#16] [❓FAQ] [answered]
What’s the retention period for backups?

**Comment by TheraniAA** [comment, marked as answer]
The retention period for backups in ChistaDATA varies depending on the service plan chosen by the customer. Generally, ChistaDATA offers daily backups and retains them for up to 7 to 30 days for Startup and Enterprise Tier respectively. However, customers can also customize the retention period in the Business Critical Tier plan to meet their specific requirements.
